.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
+install:
+- npm install
+- npm run bower:install
```

Comment:
We can solve the problem you have on a separate commit. In fact i'm already working with a grunt solution, so i can address that too. 
In my environment i have bower globally installed (npm install -g bower).